execution/scheduler_int_test.go

```diff
@@ -0,0 +1,91 @@
+package execution
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/lib"
+	"go.k6.io/k6/lib/testutils"
+	"go.k6.io/k6/lib/testutils/minirunner"
+	"go.k6.io/k6/metrics"
+)
+
+func getBogusTestRunState(tb testing.TB) *lib.TestRunState {
+	reg := metrics.NewRegistry()
+	piState := &lib.TestPreInitState{
+		Logger:         testutils.NewLogger(tb),
+		RuntimeOptions: lib.RuntimeOptions{},
+		Registry:       reg,
+		BuiltinMetrics: metrics.RegisterBuiltinMetrics(reg),
+	}
+
+	return &lib.TestRunState{
+		TestPreInitState: piState,
+		Options:          lib.Options{},
+		Runner:           &minirunner.MiniRunner{},
+		RunTags:          piState.Registry.RootTagSet().WithTagsFromMap(nil),
```

Comment:
Is it expected?

Reply:
good catch - it won't hurt, but I have no idea how that got there :sweat_smile:

Reply:
Fixed with the [latest force-push](https://github.com/grafana/k6/compare/a42bdc7a4890271f333d24dc070459bff35c6e30..9f050f7298f802e394a2b08d24e7bd40f8a4df94)